### ISSUE TITLE
Add icons to navigation menu items

### ISSUE THREAD

**Issue by TQxTim** [Flamenco/grav-plugin-editor#7]
It would be helpful to have icons for each navigation menu item. Thus users could more easily identify each category while using the the left admin navigation in collapsed mode.

Current behaviour:
![image](https://user-images.githubusercontent.com/43742253/61465563-91df2200-a978-11e9-89ba-8526f3d99ec8.png)

Button suggestions:
PHP => fa-php
JS => fa-js
CSS => fa-css3
Twig => fa-symfony
Markdown => fa-markdown
YAML => fa-hashtag

**Comment by Flamenco** [comment]
We tried this, but grav uses fa version 4, and several of those icons are only in version 5.  And the brands need a special prefix that grav ignores.

Converting to version 5 breaks all sorts of things in other plugins.

We tried this:
<img width="249" alt="Screen Shot 2019-08-28 at 3 41 30 AM" src="https://user-images.githubusercontent.com/819940/63838661-c10b8a80-c94b-11e9-8618-fa425b9d95d0.png">

But settled for this:
<img width="293" alt="Screen Shot 2019-08-28 at 4 09 27 AM" src="https://user-images.githubusercontent.com/819940/63838682-c963c580-c94b-11e9-8a7a-82a4acbd5399.png">
